client/scripts/views/pages/community_cards.tsx

```diff
@@ -78,62 +78,33 @@ class ChainCard implements m.ClassComponent<ChainCardAttrs> {
   }
 }
 
-type CommunityCardAttrs = { community: ChainInfo };
-
-class CommunityCard implements m.ClassComponent<CommunityCardAttrs> {
-  view(vnode) {
-    const { community } = vnode.attrs;
-
-    const redirectFunction = (e) => {
-      e.preventDefault();
-      localStorage['home-scrollY'] = window.scrollY;
-      m.route.set(`/${community.id}`);
-    };
+// Corresponds to 'ChainCategoryTypes' id numbering (which currently only includes DeFi and DAO)
+const filterMap = {
```

Comment:
these should be algorithmically set, have the list of ChainCategoryTypes returned for this page so we don't have to maintain the list.

Reply:
specifically, Defi + Dao